File: src/main/java/br/com/sd/entitys/Board.java
package br.com.sd.entitys;

public class Board {

    private int height;
    private int width;
    private int[][] grid;

    public Board(int height, int width) {
        this.height = height;
        this.width = width;
        this.grid = new int[height][width];

        for (int i = 0; i < height; i++) {
            for (int j = 0; j < width; j++) {
                this.grid[i][j] = 0xFFFFFF;
            }
        }
    }

    public void setPixel(int x, int y, int color) {
        this.grid[y][x] = color;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

    public String toString() {
        StringBuilder out = new StringBuilder();
        out.append("Board:\n");
        for (int i = 0; i < this.height; i++) {
            for (int j = 0; j < this.width; j++) {
                String color = String.format("%06X", this.grid[i][j]);
                if (color.charAt(0) == 'F') {
                    out.append(".");
                } else {
                    out.append(color.charAt(0));
                }
                out.append("  ");
            }
            out.append("\n");
        }

        return out.toString();
    }
}

File: pyapp/entitys/board.py
class Board:
    def __init__(self, height,  width):
        self.height = height
        self.width = width
        self.grid = []

        for i in range(height):
            for j in range(width):
                self.grid[i][j] = 0xFFFFFF

    def __init__(self, initGrid: list[list[int]], width, height):
        self.width = width
        self.height = height
        self.grid = []

        for y in range(height):
            self.grid.append([])
            for x in range(width):
                self.grid[y].append(initGrid[y][x])

    def getColorAt(self, x, y):
        return self.grid[y][x]

    def setPixel(self, x,  y,  color):
        self.grid[y][x] = color

    def getHeight(self):
        return self.height
    

    def setHeight(self, height):
        self.height = height
    

    def  getWidth(self):
        return self.width
    

    def setWidth(self, width):
        self.width = width
    

    def getGrid(self):
        gridCopy = []

        for i in range(self.height):
            for j in range(self.width):
                gridCopy[i][j] = self.grid[i][j]
        
        return gridCopy


    def toString(self):
        out = ""

        out += "Board:\n"
        for y in range(self.height):
            for x in range(self.width):
                color = hex(self.getColorAt(x, y))
                if (color[0] == 'F'):
                    out += "."
                else:
                    out += color[0]
                
                out += "  "
            
            out += "\n"
        

        return out
    

